test/classes/jade/owned/adopt-release.chpl

```diff
@@ -35,13 +35,3 @@ class A {
   delete un2;
 
 }
-
-
-{
-  writeln("test adopt ownership transfer to owned object");
-  var own1: A? = new A(2, 3);
-  var own2 = owned.adopt(own1);
-  // own1 should now be invalid/nil
-  writeln(own1);
-  writeln(own2);
-}
```

Comment:
How about keeping this snippet, replacing Line 43 with `own2 = own1;` ?

If so, let us remove "invalid" because this is not what is going on.